app/models/organization.rb

```diff
@@ -39,4 +40,22 @@ def next_reg_number_seq
     reg_number_seq
   end
 
+  def ensure_url
+    if (URI.parse(web).class == URI::HTTP || URI.parse(web).class == URI::HTTPS) 
```

Comment:
Line is too long. [81/80]<br>Don't use parentheses around the condition of an `if`.<br>Trailing whitespace detected.